aten/src/ATen/preprocess_declarations.py

```diff
@@ -217,8 +217,20 @@ def signature(option, i=None, value=None):
                                                                        (raw_args - filtered_args)]
 
 
+def is_internal_method(option):
+    if 'method' in option['variants']:
+        return False
+    elif option.get('deprecated', False):
+        return False
+    elif not option['variants']:
```

Comment:
don't we always fill in option['variants'] with at least one of 'method', 'function'?

Reply:
Actually, we need to fix the defaults for `Declarations.cwrap` too...

Reply:
yuck.